docs/geedocs/5.2.4/answer/7160003.html

```diff
@@ -0,0 +1,439 @@
+    <p><strong>Rewrite KML URLs</strong>. Users can rewrite the URLs for KML resorces included in databases at publish time.</p>
+    <p><strong>Maps API Javascript Files</strong>. Maps API Javascript files are now available and installed at <code>/opt/google/gehttpd/htdocs/maps/mapfiles</code>.</p> 
+  <h5>
+    <p id="supported_5.2.3">Supported Platforms</p>
+  </h5>
+  <p>The Open GEE 5.2.3 release is supported on 64-bit versions of the following operating systems:</p>
+    <ul>
+      <li>Red Hat Enterprise Linux version 6.x and 7.x, including the most recent security patches</li>
+      <li>CentOS 6.x and 7.x</li>
+      <li>Ubuntu 14.04 LTS and 16.04 LTS</li>
+    </ul>
+    <p>Google Earth Enterprise 5.2.3 is compatible with Google Earth Enterprise Client (EC) version 7.1.5 and above.</p>
+    <p><p>To upgrade from Open GEE 5.2.0, do NOT uninstall it. We recommend that you upgrade Open GEE 5.2.0 by simply installing Open GEE 5.2.3. Installing Open GEE 5.2.3 on top of Open GEE 5.2.0 will ensure that your PostgreSQL databases are backed up and upgraded correctly to the new PostgreSQL version used by Open GEE 5.2.3.</p></p>
+  <h5>
+    <h5>
+      <p id="resolved_issues_5.2.3">Resolved Issues</p>
+    </h5>
+    <div>
+      <table class="nice-table">
+        <tbody>
+          <tr>
+            <th>Number</th>
+            <th class="narrow">Description</th>
+            <th>Resolution</th>
+          </tr>
+          <tr>
+            <td>225</td>
+            <td>Fusion lets you create folder with space in the name</td>
+            <td>AssetManager UI only allows letters, numbers, and underscores for directory name creation</td>
+          </tr>
+	  <tr>
+	    <td>226</td>
+	    <td>Verify the wrongly spelled "aquisition" is not appearing anywhere in the product</td>
+	    <td>Fixed these spelling errors by replacing with Acquisition in multiple files.</td>
+	  </tr>
+          <tr>
+            <td>337</td>
+            <td>Add a compiler version check to scons scripts</td>
+            <td>Scons scripts ensure gcc version 4.8+ is installed before continuing with builds</td>
+          </tr>
+          <tr>
+            <td>535</td>
+            <td>DownloadTutorial.sh often is not staged properly for install</td>
+            <td>Tutorial download script is now included in the Fusion RPM</td>
+          </tr>
+          <tr>
+            <td>682</td>
+            <td>Update geconfigurepublishroot to fully correct file permissions</td>
+            <td>geconfigurepublishroot recursively sets the owner and permissions for publish root</td>
+          </tr>
+          <tr>
+            <td>731</td>
+            <td>Error in publish of SSL-enabled database</td>
+            <td>Temporary fix from 5.2.2 was found to be the best solution</td>
+          </tr>
+          <tr>
+            <td>762</td>
```

Comment:
Sorry, one more question.  It looks like issue 762 is in this file but not in the same file in the 5.2.3 directory.  I think from looking at issue #762 that it is resolved in 5.2.3.  Should this change also be made to the same file in the 5.2.3 directory or is that being done under a different issue?

Reply:
I think I got this one fixed up too now